src/mbgl/text/glyph_atlas.cpp

```diff
@@ -87,34 +79,56 @@ void GlyphAtlas::getGlyphs(GlyphRequestor& requestor, GlyphDependencies glyphDep
     }
 }
 
-void GlyphAtlas::setObserver(GlyphAtlasObserver* observer_) {
-    observer = observer_;
-}
-    
-void GlyphAtlas::onGlyphsError(const FontStack& fontStack, const GlyphRange& range, std::exception_ptr err) {
-    if (observer) {
-        observer->onGlyphsError(fontStack, range, err);
+GlyphAtlas::GlyphRequest& GlyphAtlas::requestRange(Entry& entry, const FontStack& fontStack, const GlyphRange& range) {
```

Comment:
Why does `requestRange` return the `GlyphAtlas::GlyphRequest&`? Doesn't seem like we're using it anywhere...